.github/workflows/cross-platform-builds.yml

```diff
@@ -1,9 +1,6 @@
 name: Cross-Platform Builds
 
-on:
-  push:
-    branches:
-      - main
+on: [pull_request]
```

Comment:
This change did cause the workflow to run during the PR check, but it didn't actually do anything. To fully test, you'd also need to modify line 20 here as follows. (The last param is `prBuildWorkflow`, you can simply pass in `pushBuildWorkflow` as an arg for now.)

https://github.com/ampproject/amphtml/blob/07cb943d06ae2584696ffea6d2ef457fffde7838/build-system/pr-check/cross-platform-builds.js#L13-L20

```js
runCiJob(jobName, pushBuildWorkflow, pushBuildWorkflow)
```

Reply:
Thank you for the info and for catching this! You were right of course